blogs/eks-multi-account-spire/README.md

```diff
@@ -219,13 +218,13 @@ The AWS Cloud Map namespace in the backend account is used for service discovery
 ## Deploy the SPIRE Server
 Earlier, we modified the aws-auth ConfigMap to allow the The SPIRE server in the main EKS cluster (**eks-cluster-shared**) to verify the identities of the SPIRE agents during node attestation. We also need to create copies of the [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) files for the frontend (**eks-cluster-frontend**) and backend (**eks-cluster-backend**) EKS clusters and make them available to the SPIRE server through ConfigMaps mounted as volumes.  Executing the following helper script will expedite this process: 
 ```bash
-./eks-multi-account-spire/helper-scripts/kubeconfig.sh
+./helper-scripts/kubeconfig.sh
 ```
 This script creates a spire namespace within the main EKS cluster and launches two new ConfigMaps in that namespace (**front-kubeconfig** and **back-kubeconfig**) that store copies of the kubeconfig data for the frontend and backend EKS clusters respectively.  Since the SPIRE server will be conducting cross-account cluster authentication, the kubeconfig data also specifies the ARN of the corresponding cross-account IAM role (**eks-cluster-frontend-access-role** and **eks-cluster-backend-access-role**). 
 
 Next, install the SPIRE server using a helm chart:
 ```bash
-helm install appmesh-spire-server eks-multi-account-spire/appmesh-spire-server \
+helm install appmesh-spire-server appmesh-spire-server \
```

Comment:
Where is the repo for `appmesh-spire-server`? If using eks-charts, then we should also have a step ensuring that repo exists.

Reply:
If local, then add the ./ prefix to indicate as such (not necessary but helps for readability).

Reply:
Added ./ prefix for clarity.